test/tbbmalloc/test_malloc_new_handler.cpp

```diff
@@ -33,9 +33,7 @@
 #pragma warning (disable: 4800)
 #endif //#if _MSC_VER
 
-#include "../../src/tbb/tls.h"
-
-tbb::detail::r1::tls<bool> new_handler_called;
```

Comment:
Did it cause warning as well? If so, do we have a workaround for other places where `r1::tls` is used?

Reply:
I not found any other usages of ```r1::tls```.

Reply:
This warning affect only `r1::basic_tls` and `r1::tls` with non-pointer type as template parameter.
There is no other cases of usage `r1::basic_tls` or `r1::tls` with non-pointer type.

Reply:
Thank you for the explanation.